fuels-contract/src/contract.rs

```diff
@@ -162,20 +187,6 @@ impl Contract {
         // https://github.com/FuelLabs/fuels-rs/issues/90
         // Here we generate a random coin input to prevent transaction id
         // collision in method calls that could be view-only.
```

Comment:
Oh, you forgot to delete this comment here; we don't generate the hacky random coin input anymore! 🎉

Reply:
Yeah I was so sceptical of my PR that I must have forgot haha